lib/rails_translation_manager/locale_checker/incompatible_plurals.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+class IncompatiblePlurals < BaseChecker
+  include LocaleCheckerHelper
+
+  def report
+    format_incompatible_plurals if incompatible_plurals.present?
+  end
+
+  private
+
+  def format_incompatible_plurals
+    <<~OUTPUT.chomp
+      \e[31m[ERROR]\e[0m Incompatible plural forms, for:
+
+      #{incompatible_plurals.join("\n\n")}
+
+      #{non_plural_keys_message}
+    OUTPUT
+  end
+
+  def incompatible_plurals
+    @incompatible_plurals ||= grouped_plural_keys_for_locale.flat_map do |plural_keys_for_locale|
+      locale = plural_keys_for_locale[:locale]
+
+      if all_plural_forms[locale].blank?
+        missing_plural_form_message(locale)
+      else
+        missing_plural_keys(plural_keys_for_locale).compact
+      end
+    end
+  end
+
+  def missing_plural_form_message(locale)
+    "- \e[31m[ERROR]\e[0m Please add plural form for '#{locale}' <link to future documentation>"
+  end
+
+  def non_plural_keys_message
+    "\e[1mIf the keys reported above are not plurals, rename them avoiding plural keywords: #{PLURAL_KEYS}\e[22m"
```

Comment:
Nice 👍  though I don't see much point in encapsulating this in its own method. I'd just move it inline into `format_incompatible_plurals`.